deepchecks/suites/__init__.py

```diff
@@ -9,7 +9,7 @@
 # ----------------------------------------------------------------------------
 #
 """Module contains all prebuilt suites."""
-from .integrity_suite import *
-from .methodology_suite import *
-from .performance_suite import *
-from .overall_suite import *
+from .integrity import *
```

Comment:
This may be a bit confusing given that (for some) these are also the submodules of the check, no?
(e.g. Integrity checks)
Mabye should be integrity_suites and so forth?

Reply:
It's the exact same names as with the checks, except overall that exists here. But this will change soon when we change the default suites we offer. IMO it's clear that if it's under suites then it's a suite, and also we can't have it be "integrity_suites" because that's confusing with the actual suite that is inside the module

Reply:
yes understood that double-naming is problematic, that's why offered integrity_suites
(and then inside it there will be integrity_suite)
but get your case, if it's inside the suites, then seems fine (overlap will be partial, according to the suite types...)
coming to think about it - why don't we just have a single suites file? (I know it used to be like that, and then changed because we had multiple suites of each type, but will this be the case now?)

Reply:
so I'm keeping this like that, and when we get to the reorg then it'll all be in one file